raiden/tests/conftest.py

```diff
@@ -192,7 +192,7 @@ def pytest_runtest_call(item):
             capmanager = item.config.pluginmanager.getplugin("capturemanager")
             with capmanager.global_and_fixture_disabled():
                 item.config.pluginmanager.get_plugin("terminalreporter")._tw.write(
-                    "FLAKY ", yellow=True
+                    f"FLAKY ", yellow=True
```

Comment:
superfluous `f`

Reply:
Argh, that's a zombie f-string. I already fixed that in another PR after Karl pointed it out 🙈